app-web/src/components/Search/index.js

```diff
@@ -38,18 +41,29 @@ const Container = styled.div`
     margin: 4px;
   }
 `;
-
+const AlertMessage = styled(Alert)`
+  margin: 10px auto;
+`;
 export const TEST_IDS = {
   input: 'searchbar-input',
   button: 'searchbar-button',
 };
 
 export const Search = ({ onSearch, searchOnEnter, inputConfig, query, ...rest }) => {
   const [terms, setTerms] = useState('');
+  const { authenticated } = useAuthenticated();
```

Comment:
The alert box should live outside of the search component. This will have it appear on the resource type pages /components, /documentation etc.

Reply:
Thanks, I have rebased and move this to masthead.js